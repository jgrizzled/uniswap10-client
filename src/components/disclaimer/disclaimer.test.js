import Disclaimer from '.';

const component = <Disclaimer />;

// smoke test
it('renders without crashing', async () => {
  const div = document.createElement('div');
  ReactDOM.render(<Providers>{component}</Providers>, div);
  ReactDOM.unmountComponentAtNode(div);
});

// shallow snapshot test
it('renders as expected', async () => {
  const wrapper = Enzyme.shallow(component, { wrappingComponent: Providers });
  expect(wrapper).toMatchSnapshot();
});
