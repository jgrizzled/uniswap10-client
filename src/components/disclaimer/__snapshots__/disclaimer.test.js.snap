// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<styled.div>
  <h3>
    Disclaimer
  </h3>
  <p>
    This website is for informational purposes only. This is not investment advice or an endorsement of any product or service. Index performance is hypothetical only. Data is not guaranteed to be accurate. We have no affiliation with Uniswap or any token listed on this site. Use this website at your own risk.
  </p>
</styled.div>
`;
